[Email Security] Zero Trust Email Security documentation (#15928)
cloudflare/cloudflare-docs: content/cloudflare-one/insights/email-monitoring/search-email.md

---
title: Search email

> pcx_content_type: concept

Reviewer: ```suggestion
pcx_content_type: how-to
```

weight: 2
---

# Search email

With Email Security, you can use different screen criteria to search through your email, reclassify and move a certain volume of messages, find similar emails, and export messages.

## Screen criteria

Email Security allows you to use popular, regular, and advanced screening criteria to search through your inbox. Advanced screening will give you the most in-depth investigation of your inbox.

To screen through your email traffic:

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/).
2. Select **Zero Trust**.
3. Select **Email Security**.
4. Select **Investigation**, then **Run new screen**.

> 5. Choose **Popular**, **Regular**, **Advanced** screen methods.

Reviewer: ```suggestion
5. Choose between **Popular**, **Regular**, and **Advanced** screen methods. Refer to the explanation below to learn what each method does.
```

The results will be displayed on a table. The table allows you to review and take action on the messages that match your chosen screening criteria.

### Popular screen

A popular screen allows you to view messages based on common pre-defined criteria.

To use a popular screen criteria:

1. Under **Method**, select **Popular screens**.
2. Select one of the following criteria:

>    * **Moved emails**: View emails automatically or manually moved within the last 7 days.
>    * **Reclassified emails**: Emails that had their disposition reclassified within the last 7 days.
>    * **Malicious emails**: Emails assigned the malicious disposition within the last 7 days.
>    * **Spoof emails**: Emails assigned the spoof disposition within the last 7 days.
>    * **Suspicious emails**: Emails assigned the suspicious disposition within the last 7 days.
>    * **Spam emails**: Emails assigned to the SPAM disposition within the last 7 days.

Reviewer: ```suggestion
   * **Moved emails**: View emails automatically or manually moved within the last seven days.
   * **Reclassified emails**: Emails that had their disposition reclassified within the last seven days.
   * **Malicious emails**: Emails assigned the malicious disposition within the last seven days.
   * **Spoof emails**: Emails assigned the spoof disposition within the last seven days.
   * **Suspicious emails**: Emails assigned the suspicious disposition within the last seven days.
   * **Spam emails**: Emails assigned to the spam disposition within the last seven days.
```

3. Select **Run screen**.

To modify your screening criteria, under **Active screen criteria**, select **Modify**.

### Regular screen

A regular screen allows you to investigate your inbox by inserting a term to screen across all criteria.

To use a regular screen criteria:

1. Under **Method**, select **Regular screen**.
2. Select a **Date range**.
3. Enter a keyword.
4. Select **Run screen**.

> To include all emails as part of the search, select the toggle **Include all mail** at the bottom.

Reviewer: ```suggestion
To include all emails as part of the search, enable **Include all mail**.
```

To modify your screening criteria, under **Active screen criteria**, select **Modify**.

To reset your screening criteria, select **Reset**.

### Advanced screen

> The advanced screen criteria gives you the option to narrow message results based on specific criteria. The advanced screen has several options (such as keywords, Subject keywords, sender domain, and more) to scan your inbox.

Reviewer: ```suggestion
The advanced screen criteria gives you the option to narrow message results based on specific criteria. The advanced screen has several options (such as keywords, subject keywords, sender domain, and more) to scan your inbox.
```

To use advanced screen criteria:

1. Under **Method**, select **Advanced screen**.
2. (Required) Select a date range.
3. (Optional) Fill in the other fields. All fields, except for **Subject**, must be filled with one value only.
4. Select **Run screen**.

> To include all emails as part of the search, select the toggle **Include all mail** at the bottom.

Reviewer: ```suggestion
To include all emails as part of the search, enable **Include all mail**.
```

To modify your screening criteria, under **Active screen criteria**, select **Modify**.

To reset your screening criteria, select **Reset**.

## Reclassify messages

Reclassifying messages allows you to choose the disposition of your messages if the disposition is incorrect.

To reclassify a message:

> 1. On the **Investigation** page, under **Your matching messages**, select the message you want to reclassify.

Reviewer: Is this still in the search email page?

   * **Spam**: Traffic associated with non-malicious, commercial campaigns.
   * **Bulk**: Traffic associated with [Graymail](https://en.wikipedia.org/wiki/Graymail_%28email%29), that falls in between the definitions of `SPAM` and `SUSPICIOUS`. For example, a marketing email that intentionally obscures its unsubscribe link.
   * **Clean**: Traffic not associated with any phishing campaigns.
4. Select **Save**.

To reclassify messages in bulk, select the messages you want to reclassify > **Action** > **Reclassify**.

## Move messages

Moving messages allows you to move messages to a specific folder.

To move messages:

1. On the **Investigation** page, select all the messages you want to move.
2. Select the **Action** dropdown, then select **Move**.
3. Select among one of the following folders:
   * **Inbox**: Move messages to the primary email folder.
   * **Junk email**: Move messages to the junk or spam folder.
   * **Trash**: Move messages to the trash or deleted items email folder.

>    * **Soft delete (user recoverable)**: Move messages to the user's Deleted Items folder. This option is for Microsoft only.

Reviewer: `is for Microsoft only`--> Microsoft products? Microsoft Exchange? Microsoft 365? Not clear what this means

>    * **Soft delete (user recoverable)**: Move messages to the user's Deleted Items folder. This option is for Microsoft 0365 only.

Reviewer: ```suggestion
   * **Soft delete (user recoverable)**: Move messages to the user's Deleted Items folder. This option is for Microsoft O365 only.
```

   * **Hard delete (admin recoverable)**: Delete messages from a user's inbox.
4. Select **Save**.

## Find similar emails

Each detection has an Electronic Detection Fingerprint (EDF) hash that Email Security sends to the Search API to retrieve similar detections.

To find similar detection results:

1. On the **Investigation** page, under **Your matching messages**, search for the **Similar emails** column.
2. Select the number of similar emails. Selecting the number will show you a list of similar emails.

## Export messages

With Email Security, you can export messages to a CSV file.

To export messages:

1. On the **Investigation** page, under **Your matching messages**, select **Export to CSV**.

> 2. Select **Export messages** on the pop-up message. You can only export up to 1,000 rows from the dashboard. To export all matching messages, use the API. 

Reviewer: How can users use the API to export messages? Do we have this somewhere?

## Email status

Email Security allows you to review the status and actions of each email.

To view status and actions for each email:

1. On the **Investigation** page, select the three dots.
2. Selecting the three dots will show you the following options:
   - If the email is quarantined: 
      - **View details**: Refer to [Email details](/cloudflare-one/roles-permissions/#email-details) to learn more.

>       - **View similar emails**: Find similar emails based on the ```value_edf_hash``` (Electronic Detection Fingerprint hash).

Reviewer: ```suggestion
      - **View similar emails**: Find similar emails based on the `value_edf_hash` (Electronic Detection Fingerprint hash).
```
